parse-package.go

```diff
@@ -33,44 +33,18 @@ func parsePackage(pkg *VersionedTestPackageJson) ([]PkgInfo, error) {
 
 	results := make([]PkgInfo, 0)
 	for _, target := range targets {
-		for _, test := range pkg.Tests {
-			if test.Supported == false {
-				continue
+		if target.MinSupported == "" {
```

Comment:
this must now be present. i was thinking of only including it when needed. I _think_ that's why this PR removed some files https://github.com/newrelic/node-newrelic/pull/2581

Reply:
Future readers: this is incorrect. The issue was that a target did not have a `minSupported` attribute and was not listed in the tests. When we encounter an error parsing a versioned test manifest, we omit all of the data in that manifest.